docs/ABOUT.md

```diff
@@ -1 +1,5 @@
 "Swift is an alternative for the Objective-C language that employs contemporary programming language theory concepts and strives to present a simpler syntax. Swift introduces true named parameters and retains key Objective-C concepts, including protocols, closures and categories, often replacing former syntax with cleaner versions and allowing these concepts to be applied to other language structures, like enums." -  https://en.wikipedia.org/wiki/Swift_(programming_language)
+
+Swift is a great first language to learn because it was designed to be easily readable and writable. Swift was created by Apple as an evolution of Objective-C with those goals in mind. Apple also provides lots of high-quality documentation and tools for Swift, which make it a good place to start. However, one thing to know is that the only tool available right now to code in Swift (XCode) is only available for Macs, so if you don't own one, you won't be able to work with Swift.
```

Comment:
Swift was open sourced last year and is avalible on linux https://swift.org/download/